src/Features/ShippingLabelBanner.php

```diff
@@ -152,4 +154,20 @@ private function is_supported_country( $country_code ) {
 	private function is_supported_currency( $currency_code ) {
 		return in_array( $currency_code, $this->supported_currencies, true );
 	}
+
+	/**
+	 * Return a set of shared settings for the react component. The settings can be
+	 * retrieve in component with getSetting('shippingBanner');
+	 *
+	 * @param array $settings Component settings.
+	 * @return array
+	 */
+	public function component_settings( $settings ) {
+		$active_plugins    = Onboarding::get_active_plugins();
+		$installed_plugins = Onboarding::get_installed_plugins();
+
+		$settings['shippingBanner']['isJetPackInstalled']             = in_array( 'jetpack', $installed_plugins, true );
```

Comment:
Might be a bit more of an edge case, but in the onboarding profiler and task list we have the ability to install and activate plugins client-side.

If you want to guarantee up-to-date data, using the wc-api method `getActivePlugins()` will get the most current array of active plugins.